pal/src/exception/machexception.cpp

```diff
@@ -15,16 +13,16 @@ Module Name:
 
     Implementation of MACH exception API functions.
 
-
-
 --*/
 
+#include "pal/dbgmsg.h"
+SET_DEFAULT_DEBUG_CHANNEL(EXCEPT); // some headers have code with asserts, so do this first
+
```

Comment:
I know nothing about these big changes in PAL. Did you work on these changes, or did you sync them to dotnet coreCLR latest?